modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -58,7 +58,7 @@ run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.
   dir.create(pathname, showWarnings = FALSE, recursive = TRUE)
   
   ## Convert data to format expected by pecan.ma
-  jagged.data <- lapply(trait_env$trait.data, PEcAn.MA::jagify, use_ghs = use_ghs)
+  jagged.data <- lapply(trait_env$trait.data, jagify, use_ghs = use_ghs)
```

Comment:
I think `PEcAn.MA:::jagify` (3x :) will call non exported functions

Reply:
Yes, but the R CMD check is throwing a NOTE when I use `PEcAn:::jagify` in place of `jagify`
`There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘jagify’`

Reply:
@nanu1605 we use `package::function` expression when the function here is exported, likewise we should ideally use `package:::function` expression when the function here is an internal function (not exported). Since in this case, you've unexported`jagify`, its a good call to use the latter. Let us know if the suggested changes throw a `NOTE`.

Reply:
I'm going to disagree here: Since `jagify` is defined in the same package that calls it, we don't need any namespace qualifier at all -- whether or not it's exported! Unless any tests complain, I think the code you have here is correct.

Reply:
I must have overlooked here. I thought` jagify` was throwing a note. if there's no note then I agree with @infotroph too. No need to add namespace for daughter functions. Which makes me wonder what's the exact use of `:::` expression (Since all the non-exported functions are supposedly package's own functions.

Reply:
The tests are passing but CI checks is throwing a NOTE

Reply:
@nanu1605 as suggested by @infotroph remove the namespace and then try again?

Reply:
I did the suggested changes but I don't know why the CI checks are failing. Can you help me out here @infotroph @moki1202?

Reply:
Bah, the checks are failing on a place where `jagify` gets [called from inside the `PEcAn.priors` package](https://github.com/PecanProject/pecan/blob/develop/modules/priors/R/plots.R#L149). 

@dlebauer Do you see an easy way to remove the `jagify` usage in priors? If not I'm inclined to go back to exporting it and say we're stuck with it being part of the public interface.

Reply:
@infotroph @moki1202 @dlebauer should I go back and export `jagify` ?

Reply:
@nanu1605 Yes, let's re-export `jagify`... Not exporting functions that don't need to be exported is good, but backwards compatibility, including supporting functions that were previously exported and are now depended on my other code, is arguably even more important.